doc/nrf/releases_and_maturity/migration/migration_guide_2.7.rst

```diff
@@ -134,4 +134,11 @@ Libraries
 
 This section describes the changes related to libraries.
 
+LwM2M Carrier Library
+---------------------
+
+Many event defines have received new values. If you are using the values directly in your application, you need to check the events listed in :file:`lwm2m_carrier.h`
+As an example :c:macro:`LWM2M_CARRIER_EVENT_MODEM_SHUTDOWN` was ``13`` but is now ``14``.
+The most likely place these changes are neede is :ref:`serial_lte_modem`, where it  :ref:`SLM_AT_CARRIER` is relying on the value of the defines instead of the names.
```

Comment:
```suggestion
.. toggle::

   * Many event defines have received new values.
     If you are using the values directly in your application, you need to check the events listed in :file:`lwm2m_carrier.h`.
     The most likely place these changes are needed is :ref:`serial_lte_modem` application, where :ref:`SLM_AT_CARRIER` are relying on the value of the defines instead of the names.
```